src/Aspire.AppHost.Sdk/SDK/Sdk.in.props

```diff
@@ -5,5 +5,6 @@
     the Aspire.AppHost.Sdk -->
     <SkipAspireWorkloadManifest>true</SkipAspireWorkloadManifest>
     <IsAspireHost>true</IsAspireHost>
+    <PublishAot>false</PublishAot>
```

Comment:
This isn't working for me when I try it locally. I think file-based apps get this property explicitly written to their virtual project file which would override this. We might have to move this to our targets file instead. I'm trying that in #11445

Reply:
If it works can you apply to this PR?